api/app/src/external/fhir/api.ts

```diff
@@ -64,6 +64,46 @@ export class FHIRClient extends MedplumClient {
       }
     }
   }
+
+  // HAPI FHIR specific
+  private readonly HAPI_URI = "fhir/DEFAULT";
+
+  // HAPI FHIR specific
+  async createTenant(org: { organizationNumber: number; cxId: string }): Promise<void> {
+    const url = this.getBaseUrl() + this.HAPI_URI + "/$partition-management-create-partition";
+    const payload = {
+      resourceType: "Parameters",
```

Comment:
nit: since we use this in multiple places, `"Parameters"` would be better in a const